ext_mongodb.php

```diff
@@ -227,6 +333,57 @@ public function __wakeup() : void;
 
 	<<__Native>>
 	public function selectServer(ReadPreference $readPreference): Server;
+
+/* {{{ APM */
+	public function addSubscriber( Monitoring\Subscriber $subscriber ) : void
+	{
+		$this->subscribers[] = $subscriber;
```

Comment:
Minor nit-pick, but this allows a Subscriber to be registered twice. Although `removeSubscriber()` will remove all instances of a Subscriber since it doesn't return after `unset()`, should we treat `$this->subscribers` as a set? [SplObjectStorage](http://php.net/manual/en/class.splobjectstorage.php) could be used in a pinch for this.